src/assets/content/RiverConditions.js

```diff
@@ -1,14 +1,24 @@
 export default {
     riverConditionsCharts: [
+        {
+            id: '14',
+            name: 'January 1 - March 31, 2024',
+            drupal_url: 'https://www.usgs.gov/media/videos/us-river-conditions-october-december-2023',
```

Comment:
Looks like this is the url for the previous quarter

Reply:
yeah good catch - fixed that just before you reviewed lol my bad https://github.com/DOI-USGS/vizlab-home/pull/59/commits/d439324fd61f0d860fd0cad2645b2a8c20be934b

Reply:
Actually I meant the drupal url, not the thumbnail: https://www.usgs.gov/media/videos/us-river-conditions-october-december-2023

Reply:
wow can you tell its friday? 😅 haha okay just pushed the corrected link